lua/package-info/modules/core/show.lua

```diff
@@ -12,16 +12,15 @@ local utils = require("package-info.utils")
 ---------------------------------- HELPERS ---------------------------------
 ----------------------------------------------------------------------------
 
--- Checks if opened buffer is package.json file
-local check_if_package_json = function()
+-- Checks if the open buffer refers to a non-empty package.json file
+local is_valid_package_json = function()
     local current_buffer_name = vim.api.nvim_buf_get_name(0)
```

Comment:
Since the code is pretty simple and self-explanatory I don't think we need comments here.